### ISSUE TITLE
Create basic auth example for nginx

### ISSUE THREAD

**Issue by stevvooe** [docker/distribution#460]
This is a common deployment and there are a few important points.

The key component is to add the following header at the server level, with nginx 1.7.5 or more:

```nginx
add_header: Docker-Distribution-API-Version registry/2.0 always;
```

Or, one can use the [Headers More](http://wiki.nginx.org/HttpHeadersMoreModule) module:

```nginx
more_set_header: Docker-Distribution-API-Version registry/2.0;
```

Please see #397 for some woes.

**Comment by thaJeztah** [comment]
cc @moxiegirl (I think) :smile:

**Comment by moxiegirl** [comment]
@stevvooe I think my pull #446 closes this...I wasn't aware this one was out there.

**Comment by stevvooe** [comment]
@moxiegirl That doesn't add an example with basic auth. It only includes the correct header arguments.

**Comment by Soliah** [comment]
For anyone else running into this issue with nginx, the first comment with:

> add_header: Docker-Distribution-API-Version registry/2.0 always;

has incorrect syntax. There shouldn't be a `:` after `add_header`.

**Comment by stevvooe** [comment]
Closed out with #608 and #553.
